### PR TITLE
fix(plugin-meetings): add alias in joinMeeting request body: SPARK-514797

### PR DESCRIPTION
**Pull request by stanjiawang**
# COMPLETES #< INSERT LINK TO ISSUE >
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-514797

## This pull request addresses
< DESCRIBE THE CONTEXT OF THE ISSUE >

For third party integration join, we need pass userDisplayName as alias in joinMeeting request body.

https://confluence-eng-gpk2.cisco.com/conf/pages/viewpage.action?spaceKey=WMT&title=Third-Party+Integration+Join+Flow+Support+In+UCF

## by making the following changes

< DESCRIBE YOUR CHANGES >
Add alias in joinMeeting request body

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by robstax — CHANGES_REQUESTED** [review]
i think we need to add tests to `packages/@webex/plugin-meetings/test/unit/spec/meeting/request.js`

**Review by robstax — CHANGES_REQUESTED** [review]
add test to `packages/@webex/plugin-meetings/test/unit/spec/meeting/request.js` for the new param

**Review by robstax — APPROVED** [review]

**Review by arun3528 — APPROVED** [review]

**Review by sreenara — APPROVED** [review]
Changes look good. Just a couple of comments regarding the description.

@stanjiawang please update the description correctly, the jira link needs to go at the top line (Completes #) 
Also, please describe the manual tests that were done in the ENUMERATE TESTS section. The screenshot that you have posted can go under this section too.

File: packages/@webex/plugin-meetings/src/meeting/request.ts [hunk 1]
```diff
@@ -180,6 +183,10 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       },
     };
 
+    if (alias) {
+      body.alias = alias;
+    }
```
Comment (mkesavan13) [inline packages/@webex/plugin-meetings/src/meeting/request.ts:188]: Can't we simply set it in the body itself? If it's not passed in, it'll be undefined anyway.

**Review by mkesavan13 — APPROVED** [review]
Approved with one minor comment. Also, it'll be great if we could fit this field in the Kitchen Sink app.

File: packages/@webex/plugin-meetings/src/meeting/request.ts (hunk 1 above)
Comment (stanjiawang) [reply packages/@webex/plugin-meetings/src/meeting/request.ts:188]: ```alias``` should not be sent to backend if not passed in. 

Same as ```locale``` and ```deviceCapabilities```.

**Pull request merged**